package.json

```diff
@@ -39,13 +39,16 @@
     "amqplib": "^0.10.8",
     "axios": "^1.9.0",
     "cache-manager": "^6.4.3",
+    "class-transformer": "^0.5.1",
+    "class-validator": "^0.14.2",
     "express-formidable": "^1.2.0",
     "express-session": "^1.18.1",
     "pg": "^8.15.6",
     "reflect-metadata": "^0.2.2",
     "rxjs": "^7.8.2",
     "typeorm": "^0.3.23",
-    "viem": "^2.29.1"
+    "uuid": "^11.1.0",
+    "viem": "^2.29.0"
```

Comment:
Why was `viem` downgraded?

Reply:
Jajaja it was a rebase, I chose the greater, but looks like was not the greater.